test/integration/paging.spec.ts

```diff
@@ -66,6 +181,46 @@ describe("Integration tests for Paging", () => {
         "Unexpected number of pages received."
       );
     });
+
+    it("should fail on 400 multiple pages", async () => {
+      try {
+        const result = await client.paging.getMultiplePagesFailure();
+        await client.paging.getMultiplePagesFailureNext(result.nextLink!);
+      } catch (error) {
+        assert.equal(error.statusCode, 400);
+      }
+    });
+
+    it("should fail on invalid next link URL in multiple pages", async () => {
+      try {
+        const result = await client.paging.getMultiplePagesFailureUri();
+        await client.paging.getMultiplePagesFailureUriNext(result.nextLink!);
```

Comment:
For the tests that expect a failure, we should either do an assert.fail (I think that's a real method) after the operation is invoked but before the catch, or throw an error after the operation is called (e.g. `throw new Error('Test failure')`) and check that the error.message doesn't match 'Test failure' in the catch block.

Otherwise, if the operation doesn't throw an error the assert in the catch never gets ran and the test passes (false positive).

Edit: I see you did that for the httpInfrastructure tests, I think you just need to do the same for the paging tests.

Reply:
Good catch! 😄

Reply:
Found a few other places like this and fixed as part of this PR